### ISSUE TITLE
Wrong order of const declarations

### ISSUE THREAD

**Issue by GoodDayForSurf** [tjk/vue-o2c#10]
if OptionsApi has:
```
const prop0 = 'someProp0 Value';

export default {
  components: {...},
  data() {
    return {
      someVar: someInit({
        prop1: 'prop1',
        prop2: `${prop0}/path`,
      }),
    };
  },
};
```
Result of converting is:
```
const someVar= ref(someInit({
  prop1: 'prop1',
  prop2: `${prop0}/path`,
}));

const prop0= 'someProp0 Value';
```

So value of `prop2` is `undefined/path`

**Comment by tjk** [comment]
Thank you for reporting! Will take a look. (Sorry didn't notice this sooner)

**Comment by tjk** [comment]
Once again, thanks for reporting. Fixed in [0.1.25](https://github.com/tjk/vue-o2c/releases/tag/0.1.25).
